app/subscriber/src/components/basic-search/BasicSearch.tsx

```diff
@@ -39,7 +41,7 @@ export const BasicSearch = ({ onSearch }: IBasicSearchProps) => {
   };
 
   return (
-    <styled.BasicSearch>
+    <styled.BasicSearch $inHeader={inHeader}>
```

Comment:
What's with the `$`

Reply:
It's just the way they do it on the [styled-component docs](https://styled-components.com/docs/basics#passed-props), probably just a way to differentiate between a styled component prop/ regular prop. Not required by any means though, I could take it out!

Reply:
Gives me flashbacks to jQuery lol. I am honestly not a huge fan, I think I am trying to be too "by the books". So far it has resulted in me forgetting the `$` multiple times when passing a prop.

Reply:
I like it conceptually but I would need a rule to remind me to use the `$` sign ;)

Reply:
I'll take it out for now as we don't currently use this as a standard. Doesn't make much sense unless we changed it everywhere